### ISSUE TITLE
multiple routes - Consistency between parameters in different routes

### ISSUE THREAD

**Issue by tbec** [aqandu/aqandu_live_site#47]
- getSensorData: default id = None, getTimeAggregatedData default id = "all"
-

**Comment by rosswhitaker** [comment]
Yes.  Agreed.  This is a coding consistency issue, and does not affect the function of the API (as far as I know).  Get to this later.  Do you have a preference?

**Comment by tbec** [comment]
No preference for me
